lib/internal/worker/io.js

```diff
@@ -78,7 +78,6 @@ Object.defineProperty(MessagePort.prototype, 'onmessage', {
       MessagePortPrototype.start.call(this);
     } else {
       this.unref();
-      stopMessagePort(this);
```

Comment:
this behavioral difference : between `onmessage` and `on('message')` - should we document this? it may be apparent to some class of users, but not so for others?

Reply:
We currently don’t document `onmessage` and its behaviour ourselves (but MDN does).

I’ve added a paragraph about this to the documentation, though.